css/work.css

```diff
@@ -0,0 +1,51 @@
+.work-title-container {
+    display: flex;
+    flex-direction: row;
+    padding: 40px 0px 16px 16px;
+    gap: 10px;
+    margin-left: 270px;
+    width: 900px;
+    height: 96px;
+
+}
+
+.work-title {
+
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 40px;
+
+}
+
+.work-examples {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0px;
+    gap: 24px;
+    width: 900px;
+
+    margin-left: 270px;
+}
+
+.work-item {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 16px 0px;
+    gap: 24px;
+
+    width: 900px;
+    height: 245px;
+    box-shadow: 0px 2px 0px rgba(0, 0, 0, 0.08);
```

Comment:
If this value of box shadow used more than once in your styles - use CSS variable then